<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>STL生成点云</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>231</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>采样间距</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>60</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>X方向：</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="stepXSpinBox">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>70</width>
       <height>30</height>
      </rect>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>0.200000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>50</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>um</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>80</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>um</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>60</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Y方向：</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="stepYSpinBox">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>80</y>
       <width>70</width>
       <height>30</height>
      </rect>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>0.200000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>110</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>um</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>60</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Z方向：</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="stepZSpinBox">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>110</y>
       <width>70</width>
       <height>30</height>
      </rect>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>0.200000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>150</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>切片层数：</string>
     </property>
    </widget>
    <widget class="QLabel" name="layerNumLabel">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>150</y>
       <width>60</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>260</y>
      <width>231</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>BMP像素</string>
    </property>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>60</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>X方向：</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="pitchXSpinBox">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>70</width>
       <height>30</height>
      </rect>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="singleStep">
      <double>0.001000000000000</double>
     </property>
     <property name="value">
      <double>0.114000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>50</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>um</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>80</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>um</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>60</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Y方向：</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="pitchYSpinBox">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>80</y>
       <width>70</width>
       <height>30</height>
      </rect>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="singleStep">
      <double>0.001000000000000</double>
     </property>
     <property name="value">
      <double>0.114000000000000</double>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="readSTLButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>80</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>读取STL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="toPointCloudButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>120</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>生成点云</string>
    </property>
   </widget>
   <widget class="QPushButton" name="toTXTButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>210</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>输出TXT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="toBMPButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>290</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>输出BMP</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="initStartCheckBox">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>180</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>甩点</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="rasterCheckBox">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>260</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>栅格化</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>517</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
